docs/website/docs/developers/debugging/sanitizers.md

```diff
@@ -39,6 +39,65 @@ Enabling ASan in the IREE build is a simple matter of setting the
 cmake -DIREE_ENABLE_ASAN=ON ...
 ```
 
+#### Linking to the dynamic ASan runtime
+
+You may want to use ASan when using the python bindings.
+One way to achieve this is to build Python with ASan or whatever executable is
+going to use IREE as a shared library.
+Another option is to link to the ASan runtime dynamically instead of
+linking it statically into an executable.
+
+```shell
+cmake \
+  -DIREE_ENABLE_ASAN=ON \
+  -DCMAKE_EXE_LINKER_FLAGS=-shared-libasan \
+  -DCMAKE_SHARED_LINKER_FLAGS=-shared-libasan \
+  -DCMAKE_C_COMPILER=clang-12 \
+  -DCMAKE_CXX_COMPILER=clang++-12 \
+  ...
+```
+
+Then when running things the ASan runtime will have to be preloaded.
+
+```shell
+LD_PRELOAD=/usr/lib/llvm-12/lib/clang/12.0.0/lib/linux/libclang_rt.asan-x86_64.so \
+ASAN_SYMBOLIZER_PATH=/usr/lib/llvm-12/bin/llvm-symbolizer \
+  python ...
+```
+
+If you want to run the IREE CUDA runtime driver it is likely you would need.
```

Comment:
Use `!!! tip`? (example at the end of the file.) and move to the end of this subsection given this is gpu specific?

Reply:
Done.